src/Raven.Server/Config/Categories/MemoryConfiguration.cs

```diff
@@ -32,6 +32,12 @@ public MemoryConfiguration()
         [ConfigurationEntry("Memory.LowMemoryCommitLimitInMb", ConfigurationEntryScope.ServerWideOnly)]
         public Size LowMemoryCommitLimitInMb { get; set; }
 
+        [Description("The maximum size of context to keep in the pool after dispose")]
+        [DefaultValue(32)]
+        [SizeUnit(SizeUnit.Megabytes)]
+        [ConfigurationEntry("Memory.MaxContextSizeToKeep", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
Should be `Memory.MaxContextSizeToKeepInMb`

Reply:
done